CONTRIBUTING.md

```diff
@@ -426,6 +432,14 @@ To become a Code Owner, open a PR with the following changes:
 
 Be sure to tag the existing Code Owners, if any, within the PR to ensure they receive a notification.
 
+### Emeritus roles
+
+Contributors that are stepping down or reducing their role in the project are encouraged to move to [emeritus](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#emeritus-maintainerapprovertriager). In case of long temporary absences, contributors are encouraged to let maintainers know on the CNCF Slack #otel-collector-dev channel and to mark themselves as 'Busy' on Github.
+
+In the event that a contributor steps down without prior notice, the maintainers will attempt to contact the contributor via both Github and the CNCF Slack to confirm their status. After two weeks, if the contributor is an approver or maintainer, they may be removed from the Github review auto-assignment.
```

Comment:
I get what you're saying here but "steps down" and "without prior notice" are a bit contradictory to me because "stepping down" involves explicit communication. Maybe we could say "If two or more maintainers agree that a contributor does not appear to be fulfilling the responsibilities of their role"?

Reply:
What do you think about 'becomes inactive' as Yang suggested?

Reply:
I support it